docs/conf.py

```diff
@@ -398,3 +398,27 @@ def setup(app):
     f.write(zenodo_record)
 
 print(zenodo_record)
+
+
+# -- Creating tutorials.rst ---------------------------------------------------
+
+notebook_list = []
+
+walk = os.walk("io/")
+
+for root, dirs, fnames in walk:
+    for fname in fnames:
+        if fname.endswith(".ipynb") and "checkpoint" not in fname:
+            notebook_list.append("* :doc:`" + root + "/" + fname[:-6] + "`")
+
+notebook_string = ""
+
+for notebook in notebook_list:
+    notebook_string += "\n" + notebook
+
+title = "Tutorials\n*********\n"
+
+description = "\nThe following pages contain the TARDIS tutorials:\n"
```

Comment:
```suggestion
title = "Tutorials\n*********\n"
description = "\nThe following pages contain the TARDIS tutorials:\n"
```
Newlines shouldn't be used that often - they should be used when a code block needs to be logically splitted into chunks (like it's fine to keep a newline between L421 and L423)